### ISSUE TITLE
subParts don't disappear when a parent's visibility is turned off

### ISSUE THREAD

**Issue by DrprofLuigi** [DonBruce64/MinecraftTransportSimulator#1755]
The wheel slot is set to applyAfter the mount (`$LowerBed`). When the mount's visibility is disabled, the rim disappears, but the tire is still visible.

```
{
	"pos": [0.9375, -0.1875, 1.125],
	"rot": [0, 0, 115],
	"allowParentRemoval": true,
	"isSpare": true,
	"types": ["ground_wheel_budd"],
	"defaultPart": "unuparts:unuparts_part_unu_wheel_truck_budd",
	"minValue": 0.625,
	"maxValue": 1.25,
	"applyAfter": "$LowerBed"
}
```

![2024-01-03_00 06 21](https://github.com/DonBruce64/MinecraftTransportSimulator/assets/36775575/31c8e703-4ca6-42d0-aa65-290e7de7203c)
![2024-01-03_00 08 31](https://github.com/DonBruce64/MinecraftTransportSimulator/assets/36775575/a7bc5c65-a0d6-4baa-aa41-e6a6522f3335)

**Comment by DonBruce64** [comment]
Is this a subPart, or a part defined as applyAfter the other part?  Cause a subPart is a part on a part, not something that you'd need applyAfter for.

**Comment by DrprofLuigi** [comment]
The rim is defined to applyAfter the mount, but the tire is a part on the rim.
The rim disappears, but the tire does not.

**Comment by DonBruce64** [comment]
Addressed in 22.13.0.
